helpers/deploymentConfig.ts

```diff
@@ -126,6 +126,7 @@ export const ADDRESSES: PreconfiguredAddresses = {
     rsETH: "0xfA0614E5C803E15070d31f7C38d2d430EBe68E47",
     ezETH: "0xB8eb706b85Ae7355c9FE4371a499F50f3484809c",
     weETHs: "0xE233527306c2fa1E159e251a2E5893334505A5E0",
+    pufETH: "0x6D9f78b57AEeB0543a3c3B32Cc038bFB14a4bA68",
```

Comment:
Should we also add an entry in the `assets` map?

Reply:
I'm not sure tbh. I don't see other LST assets in this map, and honestly I don't even remember why we need it in the first place. Is it for fork tests?
<img width="436" alt="image" src="https://github.com/user-attachments/assets/b6621bd3-f86c-4ae7-bd1c-bbb9770fc7c9">

Reply:
I personally like declarative configurations, that's why I updated the token config [here](https://github.com/VenusProtocol/isolated-pools/pull/446/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ffR3358-R3375) and try to follow a similar structure [in the VIP](https://github.com/VenusProtocol/vips/pull/401/files#diff-47c683d128c389a30d1b79439aafec53cf67975c9fe46ec77a007663c670d7adR15-R48). However, such configs, if unused, will get outdated at some point and will only cause confusion – here we already see that some assets are present in this mapping and others are not.

Ideally I'd want our VIP framework to fetch a specific branch or commit with the declarative configuration and then check the final state against this configuration. This way we can keep the configuration (a) closer to the contracts it's relevant to and (b) maintained. But I don't know if there's a way to do it nicely in the npm framework – seems like we'll need an infinite number of dependencies versions. A "dumb" approach would be to just fetch the config from github right in the simulation script :D